cmd/root.go

```diff
@@ -84,5 +84,8 @@ func initConfig() {
 	// If a config file is found, read it in.
 	if err := viper.ReadInConfig(); err == nil {
 		fmt.Println("Using config file:", viper.ConfigFileUsed())
+	} else {
+		fmt.Println("Error loading config ", viper.ConfigFileUsed(), ": ", err)
```

Comment:
given this behaviour maybe `log` should be used and particularly  here `log.Fatal`

Reply:
with log, I mean across the tool

Reply:
changed to log.Fatal